BrickController2/BrickController2/UI/Pages/ControllerProfilePage.xaml

```diff
@@ -67,6 +67,7 @@
                                     </Grid.ColumnDefinitions>
                                     <controls:ColorImage Grid.Column="0" Icon="{Binding ControllerEvent.EventType, Converter={StaticResource EventTypeToImage}}" Color="{DynamicResource PrimaryColor}" WidthRequest="30" HeightRequest="30" VerticalOptions="Center"/>
                                     <StackLayout Grid.Column="1" Orientation="Horizontal" VerticalOptions="Center">
+                                        <Label Text="{Binding ControllerEvent.ControllerDeviceId}" FontSize="Large" FontAttributes="Bold"/>
```

Comment:
This is little bit confusing as it e.g. shows `1Rz` without any spacing. Please have a look at it as this should support both legacy and mupltiple events. May be just add some postfix label if the associated event has dedicated gamepad: `Rz (G:2)`

Reply:
Yes you're right - but have a look at the current ControllerDeviceId in windows - it's not just an index, it's a pretty messy string (I just used that to distinguish the events).
If we would implement this feature "the right way" we would have to implement graphical representations of the controller objects with the possibility to name and handle them.
They should have names like "controller1" so you can share them between different devices...

Reply:
![Screenshot 2024-12-09 135457](https://github.com/user-attachments/assets/10800505-815e-43a0-b189-91e094fd7388)

...not really nice (on windows)

Reply:
Such ID will definitely confuse an user on Windows. How about having some logical index (stored in memory only) assigned based on this inner ID? Just an idea.

Reply:
Here is my attempt:
https://github.com/vicocz/brickcontroller2/pull/75/commits/153578c3cf8c62e0cf58a0d179eb587daa5d565a

Reply:
...and a seperator between ControllerDeviceId and EventCode
https://github.com/vicocz/brickcontroller2/pull/75/commits/6fdd66e0f9429d64423feb1944ffaa866d4781c7